### ISSUE TITLE
Documentation: Add some kind of feedback mechanism to all docs

### ISSUE THREAD

**Issue by stuclem** [vmware/vic#3369]
Per request from @corrieb, we need to add a mechanism for users to provide feedback on the docs. Automatically adding a feedback link to every page of the doc involves hacking the gitbook CSS, which is beyond my ken at this late juncture.

As a short-term solution, I propose adding a **Feedback** topic to each publication, that includes a `mailto` link.

**Comment by mdubya66** [comment]
Why can't it just take them to http://github.com/vic-product/issues?

**Comment by stuclem** [comment]
Actually, that's probably a better idea. Once we get the docs into an official VMware doc structure, then we'll automatically get the normal VMware doc feedback mechanism, that opens Bugzilla bugs and assigns them to the appropriate person. Since we're all about Github at the moment, then using Github issues makes more sense.

**Comment by stuclem** [comment]
In the end I added both - one mailto link that emails docfeedback@vmware.com, and one link to https://github.com/vmware/vic-product/issues. I included the mailto link to docfeedback@vmware.com for customers who want to feel that they are getting an official response from VMware, rather than from the community. Sending an email to docfeedback@vmware.com automatically generates a bug in Bugzilla, and routes it to me.

The feedback links appear in the SUMMARY.md and README.md files for each doc. In the generated versions, the summary appears in the left nav bar on all pages. 

Change is in https://github.com/stuclem/vic/commit/5ad7f6ddba80e14c656d51be332dd3e609ad12b8

@corrieb and @mdubya66 does this do the trick? Thanks!
